codec-http2/src/main/java/io/netty/handler/codec/http2/Http2CodecUtil.java

```diff
@@ -225,7 +225,7 @@ public ChannelPromise doneAllocatingPromises() {
                 doneAllocating = true;
                 if (successfulCount == expectedCount) {
                     promise.setSuccess();
-                    return super.setSuccess();
+                    return setSuccess();
```

Comment:
This doesn't functionally change anything right?  I'm trying to understand your `the doneAllocatingPromises accidentally calls the overridden version of setSuccess` statement.  Either way `DefaultChannelPromise.setSuccess()` is called here?